modules/oe_translation_cdt/modules/oe_translation_cdt_mock/tests/src/Unit/ServiceMockBaseTest.php

```diff
@@ -82,21 +88,45 @@ public function testPathParameters(): void {
       'parameter4' => NULL,
       'PARAMETER5' => NULL,
     ], $pathParameters);
+
+    // Check what happens if the base url contains only host.
```

Comment:
I had a lot of issues with base URLs, so I decided to test them with and without the path component.